### PR TITLE
Container field creation

### PR DESCRIPTION
**Pull request by StevieAmb**
## Branch title

-feature/container-formation-sa

## Changes made

- The containers with the buttons and the input fields were moved to the left to more closely match the comp, and two sections were created for the aside and the activity input box.

## Anything Refactored

- More sections were added, in order to make the structure more dynamic.

## Anything needed

- A border line between the aside and the new activity form box.

## Contributors

* Stevie Ambroise
* Alex Fritz
* Devon Wigle

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -81,11 +80,16 @@ main {
...
+#break {
+  float: left;
+  width: 5%;
+  background-color: red;
+}
+
 aside {
-  float: right;
-  width: 34.5%;
+  float: left;
```
Comment (alexmfritz) [inline styles.css:90]: Nice idea using the float property to align our boxes, Stevie!

File: index.html [hunk 2]
```diff
@@ -11,41 +11,51 @@
...
+                <img src="./assets/meditate.svg"/>
+                <p class="button-title">Meditate
+              </button>
+            </div>
+            <div class="activity-card">
+              <button class="activity-button" id="exercise">
+              <img src="./assets/exercise.svg">
+              <p class="button-title">Exercise
+              </button>
+            </div>
```
Comment (devonwigle) [inline index.html:36]: I like your indentation

File: index.html [hunk 3]
```diff
@@ -11,42 +11,44 @@
...
-      <h2 class="activity-title" id="new-activity">New Activity</h2>
-      <section id="user-activity-input">
-        <label class="form-text" id="category">Select a category:</label>
-        <div class="button-box">
-          <div class="activity-card">
-            <button class="activity-button" id="study">
-              <img src="./assets/study.svg">
-              <p class="button-title">Study
-            </button>
+      <section class="big-box">
```
Comment (alexmfritz) [inline index.html:14]: Let's make sure to double back on this class attribute and think of something a bit more specific to what it represents!

File: index.html [hunk 4]
```diff
@@ -40,12 +48,12 @@ <h2 class="activity-title" id="new-activity">New Activity</h2>
           <input class="user-input" id="user-seconds" type="number" name="seconds"/></label>
         </div>
         <button id="start-activity">START ACTIVITY</button>
-      </form>
+      </section>
+      <aside id="activity-log">
```
Comment (alexmfritz) [inline index.html:54]: Maybe we can line the aside class attribute to be something inline with "big-box"

**Review by devonwigle — APPROVED** [review]

**Pull request merged**